jobs/awslogs/spec

```diff
@@ -15,6 +15,7 @@ templates:
   helpers/ctl_utils.sh: helpers/ctl_utils.sh
 packages:
 - awslogs
+- awslogs-python3
```

Comment:
🤦‍♂️  ![](http://www.clixmarketing.com/blog/wp-content/uploads/2015/05/homer-computer-doh.jpg)

Reply:
no wonder it wasn't in the `/var/vcap/packages/awslogs-python3`